src/main/java/org/sagebionetworks/web/client/presenter/ProfilePresenter.java

```diff
@@ -662,6 +680,9 @@ private void loggedInCheck() {
 	
 	private void showView(Profile place) {
 		view.clear();
+		profileSynAlert.clear();
+		projectSynAlert.clear();
+		teamSynAlert.clear();

```

Comment:
Should I put this into another pull request?
On May 27, 2015 7:09 PM, "Jay Hodgson" <notifications@github.com> wrote:

> In
> src/main/java/org/sagebionetworks/web/client/presenter/ProfilePresenter.java
> <https://github.com/Sage-Bionetworks/SynapseWebClient/pull/1689#discussion_r31183622>
> :
>
> > @@ -662,6 +680,9 @@ private void loggedInCheck() {
> >  	
> >  	private void showView(Profile place) {
> >  		view.clear();
> > +		profileSynAlert.clear();
> > +		projectSynAlert.clear();
> > +		teamSynAlert.clear();
>
> (resetSynAlertWidgets)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Sage-Bionetworks/SynapseWebClient/pull/1689/files#r31183622>
> .
>